File: 2.0/javascript/contacts/removeContactListItems.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.contacts.removeContactListItems({
      // contact list id
      id: 11646003,
      // array of contact ids to remove
      contactId: [22277003, 23377003]
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch(function (err) {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/contacts/removeContactListItems.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
client.contacts.removeContactListItems(
    # contact list id
    id=11646003,
    # array of contact ids to remove
    contactId=[22277003, 23377003]
).result()
